src/content/docs/apis/nerdgraph/examples/nerdgraph-api-alert-notification-channels.mdx

```diff
@@ -852,4 +851,4 @@ mutation {
 
 [inline-fragments]: https://graphql.org/learn/queries/#inline-fragments "Inline Fragments | Queries and Mutations | GraphQL"
 
-[alert-policies]: /docs/alerts-applied-intelligence/new-relic-alerts/alert-policies/ "Alerts and applied intelligence | New Relic Documentation"
+[alert-policies]: /docs/alerts-applied-intelligence/new-relic-alerts/alert-policies/ "Alerts | New Relic Documentation"
```

Comment:
@homelessbirds this looks like our old Markdown link format. If you have time, I'd encourage you to update these.